test/wTabs.test.js

```diff
@@ -40,3 +41,159 @@ describe('tabs', () => {
     assert.ok(buttons.some((button) => button.attributes('tabindex') === '-1'));
   });
 });
+
+describe('Tabs component', () => {
+  let selectedTab;
+
+  beforeEach(() => {
+    selectedTab = ref('one');
+    const TabFixture = {
+      template: `
+      <w-tabs v-model="selectedTab">
+        <w-tab label="Tab 1" name="one" />
+        <w-tab label="Tab 2" name="two" />
+        <w-tab label="Tab 3" name="three" />
+      </w-tabs>
+    `,
+      components: { wTabs, wTab },
+      setup: () => ({ selectedTab }),
+    };
+    render(TabFixture);
+  });
+
+  it('renders the tabs', () => {
+    expect(screen.getByRole('tablist')).toBeInTheDocument();
+    expect(screen.getByRole('tab', { name: 'Tab 1' })).toBeInTheDocument();
+    expect(screen.getByRole('tab', { name: 'Tab 2' })).toBeInTheDocument();
+    expect(screen.getByRole('tab', { name: 'Tab 3' })).toBeInTheDocument();
+  });
+
+  it('renders the wunderbar', () => {
+    expect(screen.getByTestId('wunderbar')).toBeInTheDocument();
+  });
+
+  it('updates the wunderbar position on resize', () => {
+    const wunderbar = screen.getByTestId('wunderbar');
+    expect(wunderbar.style.left).toBe('0px');
+    expect(wunderbar.style.width).toBe('0px');
+    global.innerWidth = 800;
+    global.dispatchEvent(new Event('resize'));
+    waitFor(() => {
```

Comment:
Do we need to make this test asynchronous and await `waitFor`? It looks like they're awaiting `waitFor` in the [testing-library docs](https://testing-library.com/docs/dom-testing-library/api-async/#waitfor).

Reply:
It somehow works tho 🤔

Reply:
Yeah that's why I wasn't sure if we need the await 😄

Reply:
I'll leave it as it since it works 👍